tgui/packages/tgui/interfaces/ScannerGate.jsx

```diff
@@ -320,8 +320,9 @@ const ScannerGateNanites = (props, context) => {
             <NumberInput
               value={nanite_cloud}
               width="65px"
-              minValue={1}
-              maxValue={100}
+              minValue={min_cloud_id + 1}
+              maxValue={max_cloud_id}
+              step={1}
```

Comment:
Do you know if changing step to 1 does anything here? Does this prevent from fractions or? Checked tgui-core and it mostly seems to affect dragging 
![image](https://github.com/user-attachments/assets/fc050613-adbb-4db1-bd28-c8f321460869)

Reply:
https://github.com/tgstation/tgui-core/blob/0aa77815c306dccfc43cb004006711d5a66c0bfc/lib/components/NumberInput.tsx#L134

Reply:
I threw that in because dragging was broken on that input